modules/apps/batch-engine/headless/headless-batch-engine-impl/rest-openapi.yaml

```diff
@@ -0,0 +1,113 @@
+components:
+    schemas:
+        Task:
+            properties:
+                className:
+                    example: com.liferay.headless.delivery.dto.v1_0.BlogPosting
+                    type: string
+                endTime:
+                    example: 1569261245
+                    format: int64
+                    minimum: 0
+                    type: integer
+                errorMessage:
+                    example: File import failed
+                    type: string
+                executeStatus:
+                    example: INITIALIZED
+                    type: string
+                id:
+                    example: 30130
+                    format: int64
+                    minimum: 0
+                    type: integer
+                operation:
+                    example: CREATE
+                    type: string
+                startTime:
+                    example: 1569261245
+                    format: int64
+                    minimum: 0
+                    type: integer
+                version:
+                    example: v1.0
+                    type: string
+            type: object
+        body:
+            properties:
+                file:
+                    format: binary
+                    type: string
+            type: object
+info:
+    description: ""
+    license:
+        name: "Apache 2.0"
+        url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+    title: "Headless Delivery"
+    version: v1.0
+openapi: 3.0.1
+paths:
+    /import/{taskId}/:
```

Comment:
Brian is going to ask why this is not called /task/{taskId}... or import-task/importTaskId

Reply:
Changed to import-task

Reply:
👍